plugin/src/main/java/io/jenkins/plugins/analysis/core/portlets/PullRequestMonitoringPortlet.java

```diff
@@ -0,0 +1,128 @@
+package io.jenkins.plugins.analysis.core.portlets;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import com.google.gson.JsonObject;
+
+import edu.hm.hafner.analysis.Severity;
+
+import org.kohsuke.stapler.bind.JavaScriptMethod;
+import hudson.Extension;
+import hudson.model.Run;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisResult;
+import io.jenkins.plugins.analysis.core.model.ResultAction;
+import io.jenkins.plugins.monitoring.MonitorPortlet;
+import io.jenkins.plugins.monitoring.MonitorPortletFactory;
+
+/**
+ * A portlet that can be used for the pull-request-monitoring dashboard
+ * (https://github.com/jenkinsci/pull-request-monitoring-plugin).
+ * It renders an interactive sunburst diagram for each {@link ResultAction},
+ * which is registered at the current users {@link Run}.
+ *
+ * @author Simon Symhoven
+ */
+public class PullRequestMonitoringPortlet implements MonitorPortlet {
+
+    private final ResultAction action;
+
+    /**
+     * Creates a new {@link PullRequestMonitoringPortlet}.
+     *
+     * @param action
+     *              the corresponding {@link ResultAction}.
+     */
+    public PullRequestMonitoringPortlet(final ResultAction action) {
+        this.action = action;
+    }
+
+    @Override
+    public String getTitle() {
+        return action.getLabelProvider().getName();
+    }
+
+    @Override
+    @JavaScriptMethod
+    public String getId() {
+        return "warnings-ng-" + action.getResult().getId();
```

Comment:
In <https://github.com/jenkinsci/pull-request-monitoring-plugin/commit/4273a8d58824b458b3f3cf02d80664e6a23ec0d2#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R249>, you wrote

> It is recommended to use the artifact id of the plugin or parts of it as suffix.

But this is instead a prefix. Perhaps the documentation is wrong.

Reply:
Thanks. Will fix it.